### PR TITLE
ANYRUN Add Command line parameter

### PR DESCRIPTION
**Pull request by vasilydavydov**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by adi88d** [comment]
@vasilydavydov 
Please sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=31802)

File: Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py [hunk 1]
```diff
@@ -865,7 +865,7 @@ def run_analysis(args):
             files = {
                 'file': (name, open(file_path, 'rb'))
             }
-
+
```
Comment (adi88d) [inline Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py:868]: revert this

File: Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py [hunk 2]
```diff
@@ -878,6 +878,8 @@ def run_analysis(args):
             args['env_version'] = '10'
         else:
             args['env_version'] = '7'
+        if obj_type == 'file' or obj_type == 'download' :
+            args['obj_ext_cmd'] = args.get('obj_ext_cmd')
```
Comment (adi88d) [inline Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py:882]:
```suggestion
        if obj_ext_cmd and (obj_type == 'file' or obj_type == 'download'):
            args['obj_ext_cmd'] = args.get('obj_ext_cmd')
```
and add this :
`obj_ext_cmd = args.get('obj_ext_cmd')`
in line 854 under:
`obj_type = args.get('obj_type')`

**Review by adi88d — CHANGES_REQUESTED** [review]
Hi @vasilydavydov ,
Thank you for your contribution!
Good work :)
- Please bump the pack version in the pack_metadata.json file and add notes describing your changes to a new Release Notes file.
To do that, please run the `demisto-sdk update-release-notes` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.
- Please attach screenshot or pdf file for this request API documentation

**Comment by vasilydavydov** [comment]
Hello,

Pack_metadata.json and the new Release Notes file were updated in current brunch

Screenshot of documentation is here (link<https://any.run/api-documentation/#api-Analysis-PostAnalysis>)

***@***.***


Best regards,

Vasily Davydov | Product Manager | ANY.RUN

Mobile: +995 555 136 849<tel:+995555136849> | ***@***.******@***.***>



From: Adi Daud ***@***.***>
Date: Wednesday, 27 December 2023 at 14:53
To: demisto/content ***@***.***>
Cc: Vasily Davydov ***@***.***>, Mention ***@***.***>
Subject: Re: [demisto/content] ANYRUN Add Command line parameter (PR #31802)

@adi88d requested changes on this pull request.

Hi @vasilydavydov<https://github.com/vasilydavydov> ,
Thank you for your contribution!
Good work :)

  *   Please bump the pack version in the pack_metadata.json file and add notes describing your changes to a new Release Notes file.
To do that, please run the demisto-sdk update-release-notes command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.
  *   Please attach screenshot or pdf file for this request API documentation

________________________________

In Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py<https://github.com/demisto/content/pull/31802#discussion_r1436943435>:

> @@ -865,7 +865,7 @@ def run_analysis(args):

             files = {

                 'file': (name, open(file_path, 'rb'))

             }

-

+

revert this

________________________________

In Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py<https://github.com/demisto/content/pull/31802#discussion_r1436948075>:

> +        if obj_type == 'file' or obj_type == 'download' :

+            args['obj_ext_cmd'] = args.get('obj_ext_cmd')
⬇️ Suggested change

-        if obj_type == 'file' or obj_type == 'download' :

-            args['obj_ext_cmd'] = args.get('obj_ext_cmd')

+        if obj_ext_cmd and (obj_type == 'file' or obj_type == 'download'):

+            args['obj_ext_cmd'] = args.get('obj_ext_cmd')

and add this :
obj_ext_cmd = args.get('obj_ext_cmd')
in line 854 under:
obj_type = args.get('obj_type')

File: Packs/ANYRUN/ReleaseNotes/1_0_21.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### ANY.RUN
+- Added the *obj_ext_cmd* argument to **anyrun-run-analysis** command.
```
Comment (ShirleyDenkberg) [inline Packs/ANYRUN/ReleaseNotes/1_0_21.md:3]:
```suggestion
Added the *obj_ext_cmd* argument to ***anyrun-run-analysis*** command.
```

**Comment by ShirleyDenkberg** [comment]
@adi88d Doc review completed.

**Review by adi88d — APPROVED** [review]

**Pull request merged**